### PR TITLE
Feature grid: rowClassName as function and CSS key fixes

### PR DESCRIPTION
**Pull request by ahennr**
### Description:
* use kebabCase for row keys
  * If not `ol_uid` is used as row key (e.g. by using `keyFunction` prop with `keyFunction={f => f.getId()}` this can lead to errornous row class names that contain irregular characters like `.`. In order to prevent this, keys are [`"kebabCased"`](https://lodash.com/docs/4.17.5#kebabCase).
* enable rowClassName prop being a function as well

### CODE REVIEW

File: src/Grid/FeatureGrid/FeatureGrid.jsx [hunk 1]
```diff
@@ -838,9 +843,15 @@ export class FeatureGrid extends React.Component {
       ? `${className} ${this._className}`
       : this._className;
 
-    const finalRowClassName = rowClassName
-      ? `${rowClassName} ${this._rowClassName}`
-      : this._rowClassName;
+    let rowClassNameFn;
+    if (isFunction(rowClassName)) {
+      rowClassNameFn = rowClassName;
```
Comment (KaiVolland) [inline src/Grid/FeatureGrid/FeatureGrid.jsx]: I think this ommits `this._rowClassName` which might not be intentional.

**Comment by ahennr** [comment]
IMHO, `kebabCase` only should be applied to generated CSS classes but not on row keys, therefore I've left them as is. If `kebabCase` would be applied (e.g. in `getFeatureFromRowKey`) feature filtering would only work if passed `keyFunction` applies `kebabCase` to it's keys as well, or am I wrong?

File: src/Grid/FeatureGrid/FeatureGrid.jsx (hunk 1 above)
Comment (ahennr) [reply src/Grid/FeatureGrid/FeatureGrid.jsx]: Alright, I've created a function that calls passed function of prop and appends `this._rowClassName`. See 558dc531

**Comment by ahennr** [comment]
THX for your reviews, I'll merge as soon as Travis is happy.

**Comment by dnlkoch** [comment]
Go for it, you're right.

**Pull request merged**